src/test/java/org/opentripplanner/middleware/docs/SwaggerTest.java

```diff
@@ -0,0 +1,41 @@
+package org.opentripplanner.middleware.docs;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.middleware.OtpMiddlewareTest;
+import org.opentripplanner.middleware.utils.YamlUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.nio.file.Files;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class SwaggerTest extends OtpMiddlewareTest {
```

Comment:
Should this class/test be part of this PR? The test fails for me.

Reply:
Perfect segway fo me to updating swagger.

Reply:
@landonreed for the purpose of this PR, we should remove this test (and bring it back to the portion of #37 concerning swagger).

Reply:
#48 removes this test file.